test/evergreen.yml

```diff
@@ -2381,6 +2418,27 @@ tasks:
             ASAN_SYMBOLIZER_PATH=/opt/mongodbtoolchain/v4/bin/llvm-symbolizer
           format_test_script_args: -S
 
+  # FIXME-WT-8981 - Ubuntu PPC tests have been replaced by RHEL8 tests as of WT-8688, but the RHEL8 tests are currently disabled 
+  # due to platform issues. On completion of WT-8981 this can be deleted.
+  - name: format-asan-smoke-ppc-test
+    commands:
+      - func: "get project"
+      - func: "compile wiredtiger"
+        vars:
+          posix_configure_flags:
+            -DCMAKE_TOOLCHAIN_FILE=../cmake/toolchains/clang.cmake
+            -DHAVE_DIAGNOSTIC=1
+            -DWITH_PIC=1
+            -DCMAKE_BUILD_TYPE=ASan
+            -DHAVE_BUILTIN_EXTENSION_LZ4=1 -DHAVE_BUILTIN_EXTENSION_SNAPPY=1 -DHAVE_BUILTIN_EXTENSION_ZLIB=1
+      - func: "format test script"
+        # Run smoke tests, don't stop at failed tests, use default config
+        vars:
+          test_env_vars:
+            ASAN_OPTIONS="detect_leaks=1:abort_on_error=1:disable_coredump=0:unmap_shadow_on_exit=1"
+            ASAN_SYMBOLIZER_PATH=/usr/lib/llvm-6.0/bin/llvm-symbolizer
```

Comment:
I get this is temporary due to WT-8981, but should this be the v4 toolchain path? (similar for the other new tasks below)

Reply:
I think we should leave it in this case. We're intentionally not using the mongo toolchains for ASAN Ubuntu PPC jobs and this matches the usage of `../cmake/toolchains/clang.cmake` instead of v3/v4 toolchains above.

Reply:
gotcha, thanks